docs/content/_navigation.json

```diff
@@ -847,6 +846,30 @@
           }
         ]
       },
+      {
+        "title": "Dagster Pipes (Experimental)",
```

Comment:
@erinkcochran87 - would like to get your thoughts on this:

instead of having pipes section deeply nested like this: 
<img width="200" alt="Screenshot 2023-10-11 at 9 55 43 PM" src="https://github.com/dagster-io/dagster/assets/4531914/0a0f5e13-c0d8-4969-8929-56743132eb05">

should we move it up to top-level in the guides section, so it's more discoverable:
<img width="200" alt="image" src="https://github.com/dagster-io/dagster/assets/4531914/007161d0-34ec-4623-9d6d-8ea6d65776b0">

Reply:
That's fine - I had a similar thought, but decided to go with following existing structure to get it out. Go forth and move!